doc/changelog.rst

```diff
@@ -9,7 +9,24 @@ v2.10.0
 -------
 Highlights
 
-TODO: add highlights
+-  All statistics preprocessors support the same operators and have a common
+   `documentation <stat_preprocs>`__. In addition, arbitrary keyword arguments
```

Comment:
This link does not work for me

Reply:
Hopefully fixed in [f79648b](https://github.com/ESMValGroup/ESMValCore/pull/2282/commits/f79648b2e609e4c3247ce88a014922dfe335ceb6)